src/event.rs

```diff
@@ -67,6 +67,16 @@ pub enum Event<'a, T: 'static> {
         event: DeviceEvent,
     },
 
+    /// Emitted on macOS to let the application know that a file or files are requested to be opened.
+    ///
+    /// This for example happens when the user double-clicks on a file in Finder and the file's type
+    /// is associated with this application. More precisely this event is emitted when either of
+    /// `application:openFile:` or `application:openFiles:` is called on the application delegate.
+    ///
+    /// Note that to allow associating file types with an application, said application must have
+    /// an appropriate `Info.plist` file in the application bundle defining `CFBundleDocumentTypes`.
```

Comment:
Is this related to the `OpenFiles` event? Isn't that just documentation for how to get macOS to allow you to open files with your application?

Reply:
I think it is related because I don't know any other scenario for when this event is relevant than picking the application from the file browser to open a file with, and in that case the application must have an appropriate `Info.plist`, otherwise it cannot be selected. It took much longer for me than I would have liked to figure out all the stuff needed for this to work so I thought I'll save some time for others by giving a pointer for what to look up.

Reply:
Just to be clear I'm not completely against removing it, I'm just describing why we might consider keeping it.

Reply:
I just don't think we should be documenting things that are largely unrelated here. This is only the `OpenFiles` event documentation, not macOS documentation. If there's some official docs you want to link, maybe. But explaining a larger setup process to get to the point of receiving the event seems out of scope.

Reply:
Alright, I removed it.